content/github/developing-online-with-codespaces/using-codespaces-in-visual-studio.md

```diff
@@ -53,8 +53,12 @@ Connecting to {% data variables.product.prodname_github_codespaces %}  with the
 
 ### Configuring a codespace for {% data variables.product.prodname_vs %}
 
-The default codespace environment created by {% data variables.product.prodname_vs %} includes popular frameworks and tools such as .NET Core, Microsoft SQL Server, Python, and the Windows SDK. {% data variables.product.prodname_github_codespaces %} created with {% data variables.product.prodname_vs %} can be customized through a subset of `devcontainers.json` properties and a new tool called devinit, included with {% data variables.product.prodname_vs %}.
+A codespace, created with {% data variables.product.prodname_vs %}, can be customized through a new tool called devinit, a command line tool included with {% data variables.product.prodname_vs %}.
 
 #### devinit
 
-The [devinit](https://docs.microsoft.com/visualstudio/devinit/getting-started-with-devinit) command-line tool lets you install additional frameworks and tools into your Windows development codespaces, as well as run PowerShell scripts or modify environment variables. devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json), which can be added to your project for creating customized and repeatable development environments. For more information about Windows codespace configuration and devinit, see [Customize a codespace](https://docs.microsoft.com/visualstudio/ide/codespaces/customize-codespaces) in the {% data variables.product.prodname_vs %} documentation.
+[devinit](https://docs.microsoft.com/visualstudio/devinit/getting-started-with-devinit) lets you install additional frameworks and tools into your Windows development codespaces, modify environment variables, and more. 
+
+devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json), which can be added to your project for creating customized and repeatable development environments. Used with a `.devcontainer.json`, your codespace will be automatically configured upon creation.
```

Comment:
Small change to use active voice. 
I also wondered what  "Used with a `.devcontainer.json`" meant when I read this. It's explained in the page referenced in the sentence that follows this, but at this stage readers may not know. Perhaps link to the relevant section of that page here:

```suggestion
devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json). You can add this to your project if you want to create a customized and repeatable development environment. Used with a `.devcontainer.json`, your codespace will be automatically configured on creation.
```

https://docs.microsoft.com/en-us/visualstudio/ide/codespaces/customize-codespaces?view=vs-2019#running-devinit-when-creating-a-codespace

Reply:
Good suggestions. I pushed a commit with a few additional changes and a normalized URL.